integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/sortcolumns/TestSortColumns.scala

```diff
@@ -37,6 +37,12 @@ class TestSortColumns extends QueryTest with BeforeAndAfterAll {
     checkAnswer(sql("select empno from sorttable1"), sql("select empno from sorttable1 order by empno"))
   }
 
+  test("create table with no dictionary sort_columns with dictionary exclude") {
```

Comment:
Suggest to add test case for:
NumberOfNoDictSortColumns < NoDictionaryCount
It means  some no dictionary columns are in sort_columns and some no dictioanry columns are not.